$NetBSD: patch-Source_ThirdParty_skia_CMakeLists.txt,v 1.1 2026/08/30 14:25:05 wiz Exp $

Add src/core/SkPathDump.cpp to the Skia sources list. It was missing,
leaving SkPath::dump(SkWStream*, bool) undefined while Dump()/DumpHex()
in src/pathops/SkPathOpsDebug.cpp reference it. NetBSD ld keeps these
unreferenced debug helpers, so linking libwebkitgtk with --no-undefined
fails. Should be upstreamed to WebKit.

--- Source/ThirdParty/skia/CMakeLists.txt.orig
+++ Source/ThirdParty/skia/CMakeLists.txt
@@ -189,6 +189,7 @@
     src/core/SkPath.cpp
     src/core/SkPathBuilder.cpp
     src/core/SkPathData.cpp
+    src/core/SkPathDump.cpp
     src/core/SkPathEffect.cpp
     src/core/SkPathIter.cpp
     src/core/SkPathMeasure.cpp
